WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -492,14 +488,14 @@ function(task, colIdx) {
 	var width = (AjxEnv.isIE || AjxEnv.isSafari) ? "22" : "16";
 
 	// first row
-	htmlArr[idx++] = "<table width=100% class='TopRow'>";
```

Comment:
why remove the class TopRow ? Its similar to what we have it in Mail lists  (TopRow/BottomRow)